cg/cli/workflow/rnafusion/base.py

```diff
@@ -94,15 +67,14 @@ def start(
     revision: str,
     compute_env: str,
     use_nextflow: bool,
-    genomes_base: Path,
     dry_run: bool,
 ) -> None:
     """Start full workflow for CASE ID."""
     LOG.info(f"Starting analysis for {case_id}")
 
     analysis_api: RnafusionAnalysisAPI = context.obj.meta_apis["analysis_api"]
     analysis_api.prepare_fastq_files(case_id=case_id, dry_run=dry_run)
-    context.invoke(config_case, case_id=case_id, genomes_base=genomes_base, dry_run=dry_run)
+    context.invoke(config_case, case_id=case_id, dry_run=dry_run)
```

Comment:
I will move the verify_case to the config_case, but I will leave the start function as it is for now. I want to minimise changes there so I don't have to fully test it and it will be taken care in a future PR as I need to addd `start` to tomte

Reply:
Sounds good :)